relationships.md

```diff
@@ -0,0 +1,280 @@
+            "leftMaxCardinality": 2147483647,
+            "rightMinCardinality": 0,
+            "rightMaxCardinality": 2147483647,
+            "type": "relationshiptype",
+            "_links": {
+              "leftType": {
+                "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+              },
+              "rightType": {
+                "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/4"
+              },
+              "self": {
+                "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/5"
+              }
+            },
+            "_embedded": {
+              "leftType": {
+                "id": 2,
+                "label": "Person",
+                "type": "entitytype",
+                "_links": {
+                  "self": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+                  },
+                  "relationshiptypes": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2/relationshiptypes"
+                  }
+                }
+              },
+              "rightType": {
+                "id": 4,
+                "label": "OrgUnit",
+                "type": "entitytype",
+                "_links": {
+                  "self": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/4"
+                  },
+                  "relationshiptypes": {
+                    "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/4/relationshiptypes"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      {
+        "id": 589,
+        "leftId": "5a3f7c7a-d3df-419c-b8a2-f00ede62c60a",
+        "relationshipTypeId": 0,
+        "rightId": "c216201f-ed10-4361-b0e0-5a065405bd3e",
+        "leftPlace": 2,
+        "rightPlace": 1,
+        "type": "relationship",
+        "_links": {
+          "relationshipType": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/5"
+          },
+          "self": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/relationships/589"
+          },
+          "leftItem": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-b8a2-f00ede62c60a"
+          },
+          "rightItem": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/items/c216201f-ed10-4361-b0e0-5a065405bd3e"
+          }
+        },
+        "_embedded": {
+          "relationshipType": {
+            "id": 5,
+            "leftLabel": "isOrgUnitOfPerson",
+            "rightLabel": "isPersonOfOrgUnit",
+            "leftMinCardinality": 0,
+            "leftMaxCardinality": 2147483647,
+            "rightMinCardinality": 0,
+            "rightMaxCardinality": 2147483647,
```

Comment:
Again, two max values here should be null or left out

Reply:
The cardinality values have been updated